cmd/server/http/http.go

```diff
@@ -449,70 +449,68 @@ func githubWebhook(payload *payload, flowSvc *flow.Service, policySvc *policyint
 				w.WriteHeader(http.StatusOK)
 				return
 			}
-			rgx := regexp.MustCompile(`\[(.*?)\]`)
-			matches := rgx.FindStringSubmatch(payload.HeadCommit.Message)
-			if len(matches) < 2 {
-				logger.Infof("http: github webhook: no service match from commit '%s'", payload.HeadCommit.Message)
+			commitInfo, err := extractAuthorFromCommit(payload.HeadCommit.Message)
+			if err != nil {
+				logger.Infof("http: github webhook: no regex match from commit '%s'", payload.HeadCommit.Message)
 				w.WriteHeader(http.StatusOK)
-				return
```

Comment:
This return statement should stay, right? If we cannot parse the commit message to a meaningful service and author we do not know what to do?

Reply:
Yep. Fixed 👍